tools/Makefile

```
@@ -25,6 +31,16 @@ $(STRIP-2.7): | $(GCC_DIR)
 $(GCC_DIR):
 	mkdir -p $@
 
+$(IDO-5.3): | $(IDO_DIR)
+	wget https://github.com/decompals/ido-static-recomp/releases/download/v0.5/ido-5.3-recomp-ubuntu-20.04.tar.gz
```

```suggestion
	wget https://github.com/decompals/ido-static-recomp/releases/download/v0.6/ido-5.3-recomp-ubuntu-20.04.tar.gz
```
Going to want to update this to v0.6. v0.5 has a bug that left empty temp files dangling in `/tmp`.

---

If you like to live on the bleeding edge:
```suggestion
	wget https://github.com/decompals/ido-static-recomp/releases/latest/download/ido-5.3-recomp-ubuntu-20.04.tar.gz
```